onnx/reference/ops/op_if.py

```diff
@@ -64,4 +64,4 @@ def _run(
                     f"Output {i!r} (branch={branch!r}, name={names[i]!r}) is None, "
                     f"available inputs={sorted(context)}, initializers={inits}."
                 )
-        return final
+        return self._check_and_fix_outputs(final)
```

Comment:
So why do we need to call _check_and_fix_outputs here (assuming it’s called in the base class)?